mama/c_cpp/src/c/bridge/qpid/codec.c

```diff
@@ -41,7 +42,7 @@ qpidBridgeMsgCodec_pack (msgBridge      bridgeMessage,
 {
     pn_data_t*          properties      = NULL;
     pn_data_t*          body            = NULL;
-    mamaPayloadType     payloadType     = MAMA_PAYLOAD_UNKNOWN;
+    char                payloadType     = MAMA_PAYLOAD_UNKNOWN;
```

Comment:
MAMA_PAYLOAD_UNKNOWN will be an enum type leading to implied casting - should set this to a constant char (or #defined char) instead.